pixl_dcmd/tests/test_main.py

```diff
@@ -53,8 +126,49 @@ def test_remove_overlay_plane() -> None:
     assert (0x6000, 0x3000) not in ds_minus_overlays
 
 
-# TODO: Produce more complete test coverage for anonymisation
-# https://github.com/UCLH-Foundry/PIXL/issues/132
+def test_anonymisation(row_for_dicom_testing, vanilla_dicom_image: Dataset) -> None:
+    """
+    Test whether anonymisation works as expected on a vanilla DICOM dataset
+    """
+
+    orig_patient_id = vanilla_dicom_image.PatientID
+    orig_patient_name = vanilla_dicom_image.PatientName
+
+    # Sanity check: study date should be present before anonymisation
+    assert vanilla_dicom_image.StudyDate is not None
```

Comment:
Should this be in this form so that it matches our later assert?
```suggestion
    assert "StudyDate" in vanilla_dicom_image
```